tensorflow_graphics/projects/pointnet/fit.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2020 Google LLC
```

Comment:
Not quite sure you need a google copyright here :)
Try to look at what other TF core contributions from third parties used.

Reply:
My bad. I've just removed it... I'm not fussed personally. Maybe something to add to https://github.com/tensorflow/graphics/issues/302 ?